Update changes due to added missing indexes to PB schema.sql (#706)
ezsystems/developer-documentation: docs/updating/4_update_2.5.md

```bash
bin/console cache:pool:clear cache.redis
```

## Updating to 2.5.3

> This step is only **required for versions higher than 2.2 and lower than 2.5.3**.

Reviewer: maybe:
> required when updating from version higher than 2.2 and lower than 2.5.3

!!! enterprise

    This step is only **required when updating from versions higher than 2.2 and lower than 2.5.3**.

>     In case of versions lower than 2.2, please skip this step or ignore the information that  indexes from a script below already exist.

Reviewer 2: ```suggestion
    In case of versions lower than 2.2, please skip this step or ignore the information that indexes from a script below already exist.
```

>     When updating an Enterprise installation to v 2.5.3, you need to run the following script to add missing indexes:

Reviewer 2: ```suggestion
    When updating an Enterprise installation to v2.5.3, you need to run the following script to add missing indexes:
```

>     ```

Reviewer 2: ```suggestion
    ``` bash
```

>     ALTER TABLE ezpage_map_zones_pages ADD INDEX `ezpage_map_zones_pages_zone_id` (`zone_id`);

Reviewer 3: @mateuszbieniek  What about PostgreSQL? This script is for MySQL only, however since 2.5 we also support Postgres.
Hint: it might be possible to create common cross-DBMS script.

Reviewer: Hmm you are right. This should work both on MySQL on PostgreSQL
```
CREATE INDEX `ezpage_map_zones_pages_zone_id` ON ezpage_map_zones_pages(`zone_id`);
CREATE INDEX `ezpage_map_zones_pages_page_id` ON ezpage_map_zones_pages(`page_id`);
CREATE INDEX `ezpage_map_blocks_zones_block_id` ON ezpage_map_blocks_zones(`block_id`);
CREATE INDEX `ezpage_map_blocks_zones_zone_id` ON ezpage_map_blocks_zones(`zone_id`);
CREATE INDEX `ezpage_map_attributes_blocks_attribute_id` ON ezpage_map_attributes_blocks(`attribute_id`);
CREATE INDEX `ezpage_map_attributes_blocks_block_id` ON ezpage_map_attributes_blocks(`block_id`);

Reviewer 3: Nope, but close, skip ` :trollface:

Reviewer: @alongosz there will be a day when I'll become rich and famous after discovering a way how to punch someone over the internet :trollface:

Reviewer 3: > @alongosz there will be a day when I'll become rich and famous after discovering a way how to punch someone over the internet :trollface:

Face to face is cheaper (15-20 PLN one way) ;)

!!! enterprise

    This step is only **required when updating from versions higher than 2.2 and lower than 2.5.3**.
    In case of versions lower than 2.2, please skip this step or ignore the information that indexes from a script below already exist.
    

>     When updating an Enterprise installation to v2.5.3, you need to run the following script to add missing indexes:

Reviewer: I think we can skip "Enterprise" here, as it is inside `enterprise` block.
